jvm/src/Language/Java.hs

```diff
@@ -58,6 +60,13 @@ module Language.Java
   , call
   , callStatic
   , getStaticField
+  -- * Reference management
+  , push
+  , pushWithSizeHint
+  , Pop(..)
```

Comment:
Hiding the constructors isn't crucial to correctness. Users can as well use them directly instead of the `popWithValue` etc.